mmeds/html/upload_data_files.html

```diff
@@ -1,38 +1,116 @@
-<div>
-    <div class="w3-card-4 w3-padding w3-panel w3-pale-yellow w3-border">
-        <h3> Note: </h3>
-        <p>Large files may take several minutes to upload. Please stay on this page until you see a message that the upload is complete.</p>
+<div class="w3-card-4 w3-padding w3-panel w3-pale-yellow w3-border">
+    <h3> Note: </h3>
+    <p>Large files may take several minutes to upload. Please stay on this page until you see a message that the upload is complete.
+    </p>
+</div>
+
+<div class="w3-container w3-grey w3-margin-top">
+    <header class="w3-container w3-margin-top">
+        <h5><b><i class="fas fa-server" aria-hidden="true"></i> Select Upload Type </b></h5>
+    </header>
+    <div class="w3-bar w3-card-4 w3-dark-grey">
+        <button onclick="openStudyAction(event, 'QiimeUpload')" class="action-link w3-third w3-button">
+            <i class="fa fa-eye w3-xxxlarge" aria-hidden="true"></i>
+            <div class="w3-clear"></div>
+            <h4>Raw Reads (Qiime)</h4>
+        </button>
+        <button onclick="openStudyAction(event, 'OTUUpload')" class="action-link w3-third w3-button">
+            <i class="fa fa-eye w3-xxxlarge" aria-hidden="true"></i>
+            <div class="w3-clear"></div>
+            <h4>OTU Table</h4>
+        </button>
+        <button onclick="openStudyAction(event, 'LEfSeUpload')" class="action-link w3-third w3-button">
+            <i class="fa fa-eye w3-xxxlarge" aria-hidden="true"></i>
+            <div class="w3-clear"></div>
+            <h4>LEfSe Table</h4>
+        </button>
+    </div>
+
+    <hr>
+
+    <div id="QiimeUpload" class="study-action w3-container w3-row-padding w3-margin-bottom" style="display:none">
```

Comment:
Remove whatever formatting is making the input form appear as a raised box, separate from the grey background. I may be `w3-container`. I know `w3-card*` does similar but I don't see in as part of this div. Do that for all the input boxes.

Reply:
It was "w3-card-4"